### ISSUE TITLE
Incorrect text color on components with accent color on new demos

### ISSUE THREAD

**Issue by GregBrimble** [material-components/material-components-web#637]
Hi, first issue filed, so please excuse any mistakes on my part.

The [new theme demo page](https://material-components-web.appspot.com/theme.html) has the accent colour: #64DD17, which, according to the [Color Tool's accessibility algorithm](https://material.io/color/#!/?view.left=1&view.right=1&primary.color=64DD17) should have black text for legibility, not white.

This appears to be related to the fix planned https://github.com/material-components/material-components-web/issues/3.

### What MDC-Web Version are you using?
v0.10.0

### Reproduction
1. On the [Theme demo page](https://material-components-web.appspot.com/theme.html), notice the white text on components with the accent color: #64DD17.
2. This is in contradiction to the [Color Tool](https://material.io/color/#!/?view.left=1&view.right=1&primary.color=64DD17)'s recommendation: black text.

### Any other information you believe would be useful?
I believe fixing https://github.com/material-components/material-components-web/issues/3 will resolve this, but in the mean-time, it may be worth considering hard-coding into the demos, the values required to make the text black on components with the accent color (e.g. `mdc-theme--text-primary-on-accent`).

**Comment by yeelan0319** [comment]
Hi @GregBrimble ,

Thanks for the carefulness. There are few known bugs related to the new demo site including this one. What we are going to do is rolling these bugs all into #608 and fix them together.
